packages/component-library/components/index.ts

```diff
@@ -10,3 +10,5 @@ export * from "./Text"
 export * from "./Spacing"
 export * from "./Heading"
 export * from "./Box"
+export { Paragraph } from "./Paragraph"
```

Comment:
There's a bunch of types that are also exported from these components, could you instead: 
```
export * from "./Paragraph"
export * from "./Heading"
```

Reply:
Ahhh that failed before in testing - namespace clashing